scripts/initramfs/init.x86

```diff
@@ -306,38 +304,39 @@ if [ ! -z "${BOOTDELAY}" ]; then
   sleep ${BOOTDELAY}
 fi
 
-print_msg "Detecting additional removable devices"
+print_msg "Checking for a volumio rootfs update on a USB device"
 mkdir /mnt/usb
-usb_done=/dev/
-for devlink in /dev/sd*; do
-  # only checking the first partition of an additional device!
-  usb_dev=$(echo ${devlink} | sed 's/[0-9]*//g')
-  if [ ${usb_done} != ${usb_dev} ] && [ ${BOOT_DEVICE} != ${usb_dev} ]; then
-    mount -t auto $(echo ${usb_dev}1) /mnt/usb >/dev/null 2>&1
-    print_msg "Checking " $usb_dev"1 for a volumio rootfs update"
+for devlink in /dev/sd[a-z]; do
+  # do the first partition of a usb device, must be removable and not the boot device!
+  if [ $(lsblk ${dev_current} --list -no tran) == usb ] && \
```

Comment:
s/dev_current/devlink  ;-)

Reply:
now that remark is valid as well, sorry :)